### DISCUSSION TITLE
Ubuntu Installation

### DISCUSSION THREAD

**Discussion by LeGrosWinnie** [nodesource/distributions#1832] [Q&A]
Hello,
When I follow the instruction I get that :
curl -fsSL https://deb.nodesource.com/setup_23.x -o nodesource_setup.sh
sudo -E bash nodesource_setup.sh
sudo apt-get install -y nodejs
Get:1 https://deb.nodesource.com/node_23.x nodistro/main amd64 nodejs amd64 23.1.0-1nodesource1 [36.8 MB]
Preparing to unpack .../nodejs_23.1.0-1nodesource1_amd64.deb ...
Unpacking nodejs (23.1.0-1nodesource1) ...
Setting up nodejs (23.1.0-1nodesource1) ...
Processing triggers for man-db (2.12.0-4build2) ...

But then :
ubuntu@vps-05480dbb:/tmp$ node -v
v22.10.0

??? everything is 23.1.0 but -v who 22.10.0.
Thanks.

**Comment by riosje** [comment]
Hi @LeGrosWinnie did you already had installed 22?
I can't repro this, I just executed it in a clean VM and it worked.
Try deleting the current node installation and running `apt-get install nodejs -y` again
